RocketChat/Rocket.Chat.ReactNative
# chore: update expo libs

## Proposed changes
The purpose of this PR is update all our expo libraries. 
So, we updated:
* `expo-apple-authentication` to  `3.2.1`;
  * https://www.npmjs.com/package/expo-apple-authentication
  *  https://github.com/expo/expo/tree/master/packages/expo-apple-authentication
* `expo-av` to `9.2.3`;
  * https://www.npmjs.com/package/expo-av
  * https://github.com/expo/expo/tree/master/packages/expo-av  
* `expo-file-system` to `11.1.3`;
  * https://www.npmjs.com/package/expo-file-system
  * https://github.com/expo/expo/tree/master/packages/expo-file-system 
* `expo-haptics` to `10.1.0`;
  * https://www.npmjs.com/package/expo-haptics
  * https://github.com/expo/expo/tree/master/packages/expo-haptics
* `expo-keep-awake` to `9.2.0`;
  * https://www.npmjs.com/package/expo-keep-awake
  * https://github.com/expo/expo/tree/master/packages/expo-keep-awake  
* `expo-local-authentication` to `11.1.1`;
  * https://www.npmjs.com/package/expo-local-authentication 
  * https://github.com/expo/expo/tree/master/packages/expo-local-authentication
* `expo-video-thumbnails` to `5.2.1`;
  * https://www.npmjs.com/package/expo-video-thumbnails
  * https://github.com/expo/expo/tree/master/packages/expo-video-thumbnails
* `expo-web-browser` to `9.2.0`;
  * https://www.npmjs.com/package/expo-web-browser
  * https://github.com/expo/expo/tree/master/packages/expo-web-browser

## Issue(s)	
N/A

## How to test or reproduce
expo-web-browser:
- open a chat -> send a link, then:
	- open this link from the chat, check if everything works fine;
	- check if the markdown is works fine;

expo-video-thumbnails:
- open a chat -> send a video, then:
	- check if  the video is work fine;
	- on your gallery, share a video with rocket chat:
		- on Android, check if show a thumbnails;
		- on IOS, check if show the name of file;

expo-local-authentication:
- open the side-bar menu -> go to settings -> security and privacy -> screen look -> unlock with passcode, then:
	- check if create password is works fine;
	- check faceID/TouchID;
	- check Fingerprint ;

expo-haptics:
- open the side-bar menu -> go to settings -> security and privacy -> screen look -> unlock with passcode, then:
	- when you choose a password, try a wrong password and check if will get I visual effect;
- from RomView -> react to a user message, long press this reaction then:
	- check if will open a screen with this react and check if everything works fine;
expo-file-system:
- from New Server View, check if the functionality "Apply Your Certificate" still works fine;

expo-av and expo-keep-awake:
- from RomView, check if the recording audio functionality still works fine;
- from RomView, check if reproduce audio still works fine;

expo-apple-authentication:
- from New Server View -> click on "Join our open workspace", then: 
	- "create an account" -> "More options", then check if "Continue with Apple" is working;
	- "Login" -> "More options", then check if "Continue with Apple" is working;



## Screenshots
N/A

## Types of changes




## Checklist



## Further comments
N/A
